Frameworks/Foundation/NSOperationQueue.mm

```diff
@@ -1,443 +1,381 @@
-/*
-Original Author: Michael Ash on 11/9/08
-Copyright (c) 2008 Rogue Amoeba Software LLC
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Foundation/NSOperationQueue.h>
+
+#import <Foundation/NSBlockOperation.h>
+#import <Foundation/NSCondition.h>
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
-rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
-persons to whom the Software is furnished to do so, subject to the following conditions:
+#import "NSOperationInternal.h"
+#import "Starboard.h"
 
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
-Software.
+#import <atomic>
+#import <mutex>
+#import <queue>
+
+static const wchar_t* g_logTag = L"NSOperationQueue";
```

Comment:
I know this name makes more sense, but our convention everywhere else is the shouty `TAG`